cascade-merge/action.yml

```diff
@@ -0,0 +1,162 @@
+# more info on https://ledgerhq.atlassian.net/l/cp/Ar8F1nmM
+name: 'Auto Cascade Merge'
+description: 'Automatically cascade merge changes from one branch to others.'
+branding:
+  icon: 'git-pull-request'
+  color: 'purple'
+
+inputs:
+  gh-token:
+    description: 'pass GITHUB_TOKEN'
+    required: true
+  default-branch:
+    description: 'Default branch name (e.g., main or master)'
+    required: true
+    default: 'main'
+  auto-merge:
+    description: 'Whether to auto-merge the pull requests or not. true/false'
+    required: true
+    default: 'false'
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0  # fetch all history so we can push to other branches
+
+    - name: Configure Git
+      run: |
+        git config user.name "GitHub Actions"
+        git config user.email "github-actions@github.com"
+      shell: bash
+
+    - name: Create PR
+      id: create_prs
+      run: |
+        # Enables the shell to print all executed commands and exit if any command fails
+        # set -ex
```

Comment:
Maybe delete or uncomment the `set -ex` call?

Reply:
uncommenting it will show a lot of noise logs which we don't want to expose in normal usage. It's only useful in case of debug